File: javascript/pangram/pangram.js
export const isPangram = (str) => {
  const alphabet = "abcdefghijklmnopqrstuvwxyz";

  // method 1
  // const stripped = str.toLowerCase().replace(/[^a-z]/g, '');
  // const sorted = [...(new Set(stripped))].sort().join('')
  // return alphabet === sorted;

  // method 2
  const lowerCased = str.toLowerCase();
  return alphabet.split('').every(ch => lowerCased.includes(ch));
};

File: csharp/pangram/Pangram.cs
using System;
using System.Linq;

public static class Pangram
{
  private const string _alphabet = "abcdefghijklmnopqrstuvwxyz";

  public static bool IsPangram(string input)
  {
    return _alphabet.All(input.ToLower().Contains);
  }
}
